### PR TITLE
Include voter-reg-status urls in state url lookup

### PR DESCRIPTION
**Pull request by mrichards42**
Also changes the endpoint name to `election-authorities/state-urls` from `election-authorities/lookup-urls`

For the done step [card](https://www.pivotaltracker.com/story/show/165228057)

### CODE REVIEW

File: src/turbovote_http_api/handlers/election_authorities.clj [hunk 1]
```diff
@@ -32,10 +32,21 @@
...
+   [:body :authorities]
+   (fn [[authority]]
+     (prn authority)
+     (merge
+      ;; lookup urls
+      (select-keys authority [:local-election-authority-lookup-url
+                              :local-reg-authority-lookup-url])
+      ;; reg status urls
+      (-> authority
+          ;; ocd-division is [id description map-of-reg-and-voting-methods]
```
Comment (eggsyntax) [inline src/turbovote_http_api/handlers/election_authorities.clj]: Love that comment, that'll be really useful to readers.

File: src/turbovote_http_api/handlers/election_authorities.clj [hunk 2]
```diff
@@ -32,10 +32,21 @@
...
 
-(def lookup-urls-from-authority-search
-  (bifrost.i/update-in-response [:body :authorities]
-                                #(select-keys (first %) [:local-election-authority-lookup-url
-                                                         :local-reg-authority-lookup-url])))
+(def state-urls-from-authority-search
+  (bifrost.i/update-in-response
+   [:body :authorities]
+   (fn [[authority]]
+     (prn authority)
```
Comment (eggsyntax) [inline src/turbovote_http_api/handlers/election_authorities.clj]: ^ leftover I presume

Comment (mrichards42) [reply src/turbovote_http_api/handlers/election_authorities.clj]: oops, thanks for catching that!

Comment (eggsyntax) [reply src/turbovote_http_api/handlers/election_authorities.clj]: You've certainly done the same for me any number of times :laughing:

**Pull request merged**